packages/wdio-mocha-framework/src/index.js

```diff
@@ -245,6 +245,11 @@ class MochaAdapter {
                 message.passed = (params.payload.state === 'passed')
             }
 
+            if (params.payload?.parent?.title && params.payload.title.match(/^"(before|after) all?" hook/)) {
+                const hookName = params.payload.title.match(/^"(before|after) all?" hook/)[0]
```

Comment:
@mgrybyk Sure, will make the changes. I need one suggestion though. `isMochaAllHooks` is defined in `wdio-allure-reporter` package. Shall I again define it in here? (In `wdio-mocha-framework`)
I can import it from `wdio-allure-reporter` though but again it would be a dependency which might not make much sense to be in here. What do you suggest?

Reply:
Oh, my bad. No need then. Try using regex only once though

Reply:
@mgrybyk I have done the changes suggested